reactor-core/src/main/java/reactor/core/publisher/FluxOnAssembly.java

```diff
@@ -91,37 +92,40 @@ static void fillStacktraceHeader(StringBuilder sb, Class<?> sourceClass, @Nullab
 	}
 
 	@SuppressWarnings("unchecked")
-	static <T> void subscribe(CoreSubscriber<? super T> s,
-			Flux<? extends T> source,
-			@Nullable AssemblySnapshot snapshotStack) {
+	static <T> CoreSubscriber<? super T> mapSubscriber(CoreSubscriber<? super T> s,
```

Comment:
this method will need a different name (eg. `snapshotSubscriber` or `backtraceSubscriber`?)

Reply:
Maybe just `wrapSubscriber`? Since this is a method, I somehow treat the name as a verb :)